kubernetes/resource_kubernetes_ingress_v1_test.go

```diff
@@ -558,3 +637,71 @@ resource "kubernetes_ingress_v1" "test" {
   wait_for_load_balancer = true
 }`, name, name, name, name, name, name, name)
 }
+
+func testAccKubernetesIngressV1Config_ruleHostOnly(name string) string {
+	return fmt.Sprintf(`resource "kubernetes_ingress_v1" "test" {
```

Comment:
Since we're using the ` (backtick) string literal, I think it would improve readability if we keep the HCL code formated as it would be tf files. That means moving the HCL part of this line on a new one below.

Reply:
Thanks, @alexsomesan. I've updated all HCL parts in this file for consistency and I will follow the same approach in my future PRs. 👍🏻